panelapp/panels/models/genepanelsnapshot.py

```diff
@@ -481,7 +487,14 @@ def _get_stats(self, use_db=True):
         out = {"gene_reviewers": [], "str_reviewers": [], "region_reviewers": []}
 
         for key in keys:
-            out[key] = out.get(key, 0) + info.get(key, 0)
+            if key not in ['str_reviewers', 'region_reviewers']:
+                out[key] = out.get(key, 0) + info_genes.get(key, 0)
+
+            if key not in ['gene_reviewers', 'region_reviewers']:
+                out[key] = out.get(key, 0) + info_strs.get(key, 0)
+
+            if key not in ['str_reviewers', 'gene_reviewers']:
+                out[key] = out.get(key, 0) + info_regions.get(key, 0)
 
```

Comment:
Perhaps an else condition here on what `out[key]` should be?

Reply:
it shouldn't be there if it doesn't satisfy the condition